_inc/lib/debugger/class-jetpack-cxn-test-base.php

```diff
@@ -450,9 +462,11 @@ public function output_fails_as_wp_error( $type = 'all', $group = 'all' ) {
 
 		foreach ( $fails as $result ) {
 			$code    = 'failed_' . $result['name'];
-			$message = $result['message'];
+			$message = $result['short_description'];
 			$data    = array(
-				'resolution' => $result['resolution'],
+				'resolution' => $result['action'] ?
+					$result['action_label'] . ' :' . $result['action'] :
+					'unknown',
```

Comment:
Is there a reason we would want to expose "unknown"? I feel we'd not want to have a resolution displayed in this case.

Reply:
good call - i think we should leave it empty if there is no action